commands.pm

```diff
@@ -170,11 +170,18 @@ sub upload_file {
         mkdir($target) or die "$!";
     }
 
-    my $upname = basename($self->param('filename'));
+    my $upname   = $self->param('upname');
+    my $filename = basename($self->param('filename'));
+    # Only renaming the file if upname parameter has posted ie. from upload_logs()
+    # With this it won't renamed the file in case upload_assert and autoyast profile
+    # as those are not called from upload_logs.
+    if ($upname) {
+        $filename = $upname;
```

Comment:
it was basenamed in upload_logs() https://github.com/os-autoinst/os-autoinst/pull/406/files#diff-bd61f4009ce05bc0647daf1170d6facaL394 , we just basename it twice here like filename param. but updated.